libfwupdplugin/fu-udev-device.c

```diff
@@ -325,6 +325,21 @@ fu_udev_device_probe_serio(FuUdevDevice *self, GError **error)
 	return TRUE;
 }
 
+static guint16
+fu_udev_device_get_property_as_uint16(GUdevDevice *udev_device, const gchar *key)
+{
+	const gchar *tmp = g_udev_device_get_property(udev_device, key);
+	guint64 value = 0;
```

Comment:
if you're capping the stoull call at G_MAX_UINT16, this should probably be a guint16.

Reply:
nah, we pass by ref, and casting a different uint type isn't always portable.